schema.json

```
@@ -219,10 +219,13 @@
       "type": "object",
       "properties": {
         "path": { "$ref": "#/$defs/path" },
-        "language": { "type": "string" },
+        "language": {
+          "description": "Explicitly map file to language.\n\nIf not provided, backend will try to detect language from language name or it's content.",
```

```suggestion
          "description": "Explicitly map file to language.\n\nIf not provided, Tolgee will try to detect language from language name or it's content.",
```
Not sure if it will be clear what "backend" for the user.